File: src/main/java/no/nav/syfo/domain/dto/Arbeidssituasjon.java
package no.nav.syfo.domain.dto;

public enum Arbeidssituasjon {
    NAERINGSDRIVENDE ("selvstendig næringsdrivende"),
    FRILANSER ("frilanser"),
    ARBEIDSTAKER ("arbeidstaker");

    private final String name;

    Arbeidssituasjon(final String name) {
        this.name = name;
    }

    public String toString() {
        return name;
    }
}

File: src/main/kotlin/no/nav/syfo/domain/dto/Arbeidssituasjon.kt
package no.nav.syfo.domain.dto

enum class Arbeidssituasjon(var navn: String) {
    NAERINGSDRIVENDE("selvstendig næringsdrivende"),
    FRILANSER("frilanser"),
    ARBEIDSTAKER("arbeidstaker");

    override fun toString(): String {
        return navn
    }
}
